src/welcome/ProgressiveProfiling.jsx

```diff
@@ -73,12 +73,14 @@ const ProgressiveProfiling = (props) => {
     e.preventDefault();
     const authenticatedUser = getAuthenticatedUser();
     const payload = { ...values, extendedProfile: [] };
-    extendedProfile.forEach(fieldName => {
-      if (values[fieldName]) {
-        payload.extendedProfile.push({ fieldName, fieldValue: values[fieldName] });
-      }
-      delete payload[fieldName];
-    });
+    if (Object.keys(extendedProfile).length !== 0) {
```

Comment:
Shouldn’t the loop don't execute if the object is empty? Why this check is necessary here?

Reply:
when object is empty its shows error that why i have add this check.

Reply:
I believe it can only happen if the `extendedProfile` is undefined or `NULL` but not an empty object `{}`. NVM please update the condition to `if (Object.keys(extendedProfile).length > 0)`.